src/index.js

```diff
@@ -4,14 +4,21 @@ import './index.css'
 import App from './App'
 import reportWebVitals from './reportWebVitals'
 import { BrowserRouter } from 'react-router-dom'
+import { AuthContextProvider } from './context/AuthContext'
+import { ChatContextProvider } from './context/ChatContext'
 
 const root = ReactDOM.createRoot(document.getElementById('root'))
 root.render(
-  <React.StrictMode>
-    <BrowserRouter>
-      <App />
-    </BrowserRouter>
-  </React.StrictMode>
+  <AuthContextProvider>
+    <ChatContextProvider>
```

Comment:
Does the chat context need to be this low in the component tree?
I think you only use this in the chat page.